draft-ietf-quic-qpack.md

```diff
@@ -821,10 +828,13 @@ connection error of type `HTTP_QPACK_DECODER_STREAM_ERROR`.
 
 ## Header Block Representations
 
-Header blocks contain compressed representations of header lists and are carried
```

Comment:
Representations of header lists?  Representations of header fields?  Also, plural hides that blocks and lists have a one-to-one correspondance.  These are my motivations to reword.